### ISSUE TITLE
SCD41 single shot low power option

### ISSUE THREAD

**Issue by hopkapi** [adafruit/Adafruit_CircuitPython_SCD4X#5]
The SCD4**1** has a single shot low power option mentioned [in the datasheet](https://www.sensirion.com/fileadmin/user_upload/customers/sensirion/Dokumente/9.5_CO2/Sensirion_CO2_Sensors_SCD4x_Datasheet.pdf) that doesn't seem to have made it into the library just yet, so as [suggested by Mikey on your CS forums](https://forums.adafruit.com/viewtopic.php?f=19&t=182186#p885549) I'm requesting you consider adding it, thanks.

**Comment by KeithTheEE** [comment]
Double checking, are you just wanting the single shot low power feature, or is there another aspect that you need for your workflow?

**Comment by hopkapi** [comment]
Yeah, just that thanks, though a bit unsure what other aspect I'd need?

**Comment by KeithTheEE** [comment]
I was just making sure there wasn't another function you needed right off the bat, I'm going to try and add it tonight or tomorrow to the library.
